### ISSUE TITLE
NY Courts

### ISSUE THREAD

**Issue by flooie** [freelawproject/courts-db#42]
The remainder for the NY Import.  Hopefully can be finished up today.  This represents about 151 unique opinions that have yet to be imported.  Half need to be added to CL and the other half needs to be updated in Courts-DB.

Court Strings
---------------

- [x] Judge's Chambers
- [x] United States District Court
- [x] New York City Court Special Term
- [x] District Court
- [x] New York City Court of Special Sessions
- [x] United States Circuit Court of Appeals
- [x] United States Circuit Court
- [x] New York Supreme CClaim No. 21807 <-- Opinionated fix
- [x] Circuit Court U. S. NEW YORK
- [x] Ontario County District Court
- [x] U.S. Circuit Court
- [x] District Court United States

IDs
-----

- [x] superctsc
- [x] circtdpa
- [x] circtmissterr
- [x] nyspecsessyonk
- [x] nyjustctwebster
- [x] nyjustctwesbste <-- this looks like a malformed string needing fix

**Comment by flooie** [comment]
Some of these were too vague and need hand checks to verify that the for instance District Court refers to lets say ny district court before there were multiple ones (just a guess)

**Comment by flooie** [comment]
Ontario County District Court 

Is a particularly weird one, because as a far as a I can tell from research the only district courts to exist are in Long Island, in Suffolk (and maybe a neighboring county).  

(It appears to be incorrect data.  The string in the PDF is (County Court, Ontario County, June 8, 2004)

**Comment by flooie** [comment]
More General Notes.

US Circuit Court (generic)
and 
US District Court (generic) 

have been added to Courts-db.  Generally these are going to relate to pre-1900 courts.

There are some oddities in the federal circuits and districts that are worth mentioning (thanks [Ballotopedia](https://ballotpedia.org/United_States_court_reorganization_legislation) for this.

<img width="793" alt="Screen Shot 2022-02-16 at 10 30 36 AM" src="https://user-images.githubusercontent.com/6464529/154298403-dc8cf335-253c-495a-a1e9-7b54ced9346a.png">

During a check of the Circuit Courts here, I assumed the circuit court would reference NY Circuits but it referenced Massachusetts.  And others referenced NY.  So I made a parent court for all the districts and circuits.  Similar to what we see on Lexis.

Also of note:
**Judges Chambers**, was a decision by a judge in South Carolina.  I was only able to track that down because he was a politician that only served as a Judge once in the Superior Court of South Carolina.
